src/Domain/CoordinateMaps/CoordinateMap.hpp

```diff
@@ -212,7 +212,7 @@ class CoordinateMap
   CoordinateMap& operator=(CoordinateMap&& /*rhs*/) = default;
   ~CoordinateMap() override = default;
 
-  constexpr explicit CoordinateMap(Maps... maps);
```

Comment:
Because for a function to be `constexpr` the compiler needs the full definition at the call site, not just a forward declaration. Since we can't really create the maps at compile time anyway I'd rather save on compile time by moving definitions into `tpp`/`cpp` files than having them be `constexpr` when they're not useful. Does that make sense?